server/sonar-server/src/main/java/org/sonar/server/computation/snapshot/Snapshot.java

```diff
@@ -28,17 +29,23 @@
 public class Snapshot {
 
   private long id;
+  private String uuid;
   private long createdAt;
 
   private Snapshot(Builder builder) {
     this.id = builder.id;
+    this.uuid = builder.uuid;
     this.createdAt = builder.createdAt;
   }
 
   public long getId() {
     return id;
   }
 
+  public String getUuid() {
```

Comment:
if I correctly understand, the column SNAPSHOTS.UUID already existed but the DTO was not up-to-date ?